doc/manual/manual.tex

```diff
@@ -6153,7 +6153,7 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 \label{sec:benchmark-van-keken}
 
 \textit{This section is a co-production of Cedric Thieulot, Juliane Dannberg,
-Timo Heister and Wolfgang Bangerth.}
+Timo Heister and Wolfgang Bangerth with an extension to this benchmark provided by the Virginia Tech Dept. of Geosciences class Geodynamics and ASPECT co-taught by Scott King and D. Sarah Stamps.}
```

Comment:
latex is complicated -- please write the last name as
```
  D.~Sarah Stamps
```
(note the tilde). Latex will otherwise believe that the period ends a sentence and add an extra-long space after it, rather than the usual inter-word spacing.